UniWatch/Controllers/ClassController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using UniWatch.DataAccess;
+using UniWatch.Models;
+
+namespace UniWatch.Controllers
+{
+    public class ClassController : Controller
+    {
+        private IClassManager _classManager;
+
+        public ClassController()
+        {
+            _classManager = new ClassManager();
+        }
+
+        //GET: Index
+        public ActionResult Index()
```

Comment:
Josh, I can't see your index. Did you create a view for your Index action on your class controller?